core/arch/arm/arm.mk

```diff
@@ -140,6 +140,15 @@ ifeq ($(CFG_ARM32_core),y)
 # CFG_DT_ADDR:       if defined, forces Device Tree data physical address.
 endif
 
+# CFG_MAX_CACHE_LINE_SHIFT is used to define platform specific outer cache
```

Comment:
`s/outer/max/`

Reply:
I am wondering is it good idea to hide the outer cache term as it is somewhat commonly used in the domain. If I try to rephrase it so that we have both terms in here.

Reply:
Updated the comment